code/modules/client/preference/preferences.dm

```diff
@@ -1850,9 +1882,12 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 							if(!(S.bodyflags & ALL_RPARTS))
 								third_limb = "r_arm"
 
-					var/new_state = input(user, "What state do you wish the limb to be in?") as null|anything in valid_limb_states
-					if(!new_state) return
+					if(!no_amputate)	// I don't want this in my menu if it's not an option, heck.
+						valid_limb_states += "Amputated"
```

Comment:
So that it only becomes added if a limb _can_ be Amputated.
. . . sorry, I was testing this code a lot and seeing "Amputated" come up over and over again for my head and torso as an IPC was grating.